code/modules/research/designs/medical_designs.dm

```diff
@@ -507,8 +507,8 @@
 //Cybernetic organs
 
 /datum/design/cybernetic_liver
-	name = "Cybernetic Liver"
-	desc = "A cybernetic liver"
+	name = "Basic Cybernetic Liver"
+	desc = "A basic cybernetic liver"
```

Comment:
i think you could do better than this, skog!

Reply:
Fairly sure I cant. Whats your suggestion?